src/plugins/discover/public/components/discover_grid/copy_column_name_button.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiCopy, EuiListGroupItemProps } from '@elastic/eui';
```

Comment:
Seems this should already work 🥳 
```suggestion
import { copyToClipboard, EuiListGroupItemProps } from '@elastic/eui';
```